weather_mv/loader_pipeline/pipeline.py

```diff
@@ -207,9 +215,10 @@ def get_coordinates(ds: xr.Dataset) -> t.Iterator[t.Dict]:
     # Example:
     #   {'longitude': -108.0, 'latitude': 49.0, 'time': '2018-01-02T23:00:00+00:00'}
     idx = 0
+    total_coords = _prod(ds.coords.dims.values())
```

Comment:
minor note: dims may not give you the right set, 
IIRC indexes was a subset of dims (check one of the grib test files.)

Reply:
Aren't the `dims` of the `coords` the correct subset? I just ran a small experiment: 
```
>>> ds = xr.open_dataset(gb, engine='cfgrib')  # gb is a path to test_data_grib_single_timestamp
>>> [len(x) for x in ds.coords.indexes.values()]
[1801, 3600]
>>> ds.coords.dims.values()
ValuesView(Frozen({'latitude': 1801, 'longitude': 3600}))
```